src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/IPGlobalProperties.cs

```diff
@@ -9,12 +10,7 @@ public abstract class IPGlobalProperties
     {
         public static IPGlobalProperties GetIPGlobalProperties()
         {
-            return new SystemIPGlobalProperties();
-        }
-
-        internal static IPGlobalProperties InternalGetIPGlobalProperties()
-        {
-            return new SystemIPGlobalProperties();
+            return IPGlobalPropertiesPal.GetIPGlobalProperties();
```

Comment:
This is one of the Pal redirections I mentioned. This is the general mechanism for returning a specialized subtype per-platform.